lib/vector-buffers/src/test/common/messages.rs

```diff
@@ -133,6 +137,10 @@ impl FixedEncodable for SizedRecord {
         buffer.advance(buf_len as usize);
         Ok(SizedRecord(buf_len))
     }
+
+    fn encoded_size(&self) -> Option<usize> {
+        Some(self.0 as usize + 4)
```

Comment:
Same comment/question here.

Reply:
Yeah, totally fair point. I'll make this a bit more robust.